Doc/whatsnew/3.12.rst

```diff
@@ -261,6 +261,12 @@ asyncio
   yielding tasks.
   (Contributed by Kumar Aditya in :gh:`78530`.)
 
+calendar
+--------
+
+* Add enums for month and day. For example: ``JANUARY`` and ``MONDAY``.
```

Comment:
This looks like global variables, not clear they are enum members.
What about making links and letting people go see the values:

```suggestion
* Add enums :data:`~calendar.Month` and :data:`~calendar.Day`.
```